src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -371,11 +352,7 @@ function WorkspaceMembersPage(props) {
                     !_.isEmpty(policyMember.errors),
                 text: props.formatPhoneNumber(PersonalDetailsUtils.getDisplayNameOrDefault(details)),
                 alternateText: props.formatPhoneNumber(details.login),
-                rightElement: isAdmin ? (
-                    <View style={[styles.badge, styles.peopleBadge]}>
-                        <Text style={styles.peopleBadgeText}>{props.translate('common.admin')}</Text>
-                    </View>
-                ) : null,
+                rightElement: roleBadge,
```

Comment:
```suggestion
                    <Badge
                        text={isOwner ? props.translate('common.owner') : props.translate('common.admin')}
                        textStyles={styles.textStrong}
                        badgeStyles={[
                            styles.alignSelfCenter,
                            styles.badgeBordered,
                        ]}
                    />
```

we can use the new badge like this, we can also avoid creating a separate variable for it.

Reply:
https://github.com/Expensify/App/blob/29ad8787c7c889271c117dd4420109f9be8e180d/src/components/MenuItem.tsx#L574-L583